com.avaloq.tools.ddk.xtext.valid.generator/src/com/avaloq/tools/ddk/xtext/valid/generator/ValidValidatorFragment.java

```diff
@@ -224,19 +224,6 @@ public static String getPreferencePackage(final Grammar grammar, final Naming na
     return naming.basePackageUi(grammar) + ".preferences"; //$NON-NLS-1$

```

Comment:
seems like this method is not needed either - or am I missing something?

Reply:
Removed.